package.json

```diff
@@ -21,6 +21,8 @@
   },
   "homepage": "https://github.com/BlueOcean35/ProjectVroom#readme",
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "material-ui-core": "^5.0.1",
```

Comment:
Might want to remove the material-ui-core and material-ui-icons that I imported (oops!)